tests/test_algorithm.py

```diff
@@ -1555,13 +1558,29 @@ def handle_data(context, data):
                 for val in sublist]
 
             self.assertEqual(len(all_txns), 67)
-            first_txn = all_txns[0]
+            # all_orders are all the incremental versions of the
+            # orders as each new fill comes in.
+            all_orders = [order for orders in
```

Comment:
I think this is just `toolz.concat(results['orders'])`:

```
(Pdb++) all_orders == list(toolz.concat(results['orders']))
True
```
(`toolz.concat` is just a more-clearly-named `itertools.chain.from_iterable`).